### PR TITLE
Romina's tutorial

### PR DESCRIPTION
**Pull request by RomiP**
new files for participants for OOP tutorial.
added tutorial to _toc.yml

### CODE REVIEW

File: book/_toc.yml [hunk 1]
```diff
@@ -46,6 +46,11 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+        - file: tutorials/OOP
```
Comment (scottyhq) [inline book/_toc.yml]: jupyterbook expects a file here, so I recommend a simple `tutorials/OOP/index.md` that just has a title: 
```
# Object Oriented Programming
```

**Review by scottyhq — CHANGES_REQUESTED** [review]
thanks @RomiP please 'restart kernel and clear outputs' for these notebooks if you want them automatically executed and rendered, otherwise you'll have to add them here: 

https://github.com/snowex-hackweek/website2022/blob/29fc416ec7a24d5d602b0773ed0706b02cf76afe/book/_config.yml#L39-L40

File: book/team.yaml [hunk 2]
```diff
@@ -348,7 +348,7 @@ people:
       Romina is a PhD candidate in atmospheric physics at the University of Toronto with a
       background in computer science and mathematics. Her research focuses on deep
       convection in the Labrador Sea and its role in atmospheric carbon sequestration.
-      In additio to her research, Romina is actively involved in a software development
+      In addition to her research, Romina is actively involved in a software development
```
Comment (JessicaS11) [inline book/team.yaml]:
```suggestion
      In addition to her research, Romina is actively involved in software development
```

**Review by scottyhq — APPROVED** [review]
thanks @RomiP !

**Pull request merged**